Library/Homebrew/cmd/update-bash.sh

```diff
@@ -36,26 +37,59 @@ git_init_if_necessary() {
     git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
   fi
 
-  if git remote show origin -n | grep -q "mxcl/homebrew"
+  if [[ "$(git remote show origin -n)" = *"mxcl/homebrew"* ]]
   then
     git remote set-url origin https://github.com/Homebrew/homebrew.git
     git remote set-url --delete origin ".*mxcl\/homebrew.*"
   fi
 }
 
+rename_taps_dir_if_necessary() {
+  local tap_dir
+  for tap_dir in "$HOMEBREW_LIBRARY"/Taps/*
+  do
+    [[ -d "$tap_dir/.git" ]] || continue
+    local tap_dir_basename
+    tap_dir_basename="$(basename "$tap_dir")"
```

Comment:
Would `"${tap_dir##*/}"` be better because it avoids calling an external command? As far as I know, there are no corner cases where the two would behave differently. But maybe using `basename` is less cryptic …